### DISCUSSION TITLE
Possible file path locations to `tts.exe`

### DISCUSSION THREAD

**Discussion by howardbaik** [coqui-ai/TTS#2566] [General Q&A]
I'm writing a function that finds the path to the local `tts.exe` file. On my Mac, I see the output of  which tts to be `/opt/homebrew/Caskroom/miniforge/base/bin/tts` . 

What are possible file path locations to `tts.exe` on Mac/Linux/Windows?

**Comment by erogol** [comment]
it is python defaults on the installed system. We don't set specific path.

**Reply by howardbaik** [reply]
Would this be a good starting point for the possible file path locations to `tts`?

 `Linux` = `/usr/bin/tts`, `/usr/local/bin/tts`
  `Darwin` = `/opt/homebrew/Caskroom/miniforge/base/bin/tts`
  `Windows` = `C:/Program Files/tts`
